browser/ui/views/tabs/tab_drag_controller.cc

```diff
@@ -327,6 +330,46 @@ gfx::Rect TabDragController::CalculateDraggedBrowserBounds(
   return bounds;
 }
 
+[[nodiscard]] TabDragController::Liveness TabDragController::ContinueDragging(
+    const gfx::Point& point_in_screen) {
+  if (!base::FeatureList::IsEnabled(tabs::features::kBraveSplitView)) {
+    return TabDragControllerChromium::ContinueDragging(point_in_screen);
+  }
+
+  auto weak = weak_factory_.GetWeakPtr();
+  const auto liveness =
+      TabDragControllerChromium::ContinueDragging(point_in_screen);
+
+  if (!weak) {
+    // In DragBrowserToNewTabStrip() could delete itself, so we need to check
```

Comment:
I'm confused, where does `DragBrowserToNewTabStrip` get called?

Reply:
In TabDragControllerChromium::ContinueDragging(point_in_screen);